packages/zosjobs/src/doc/response/IJob.ts

```diff
@@ -34,6 +34,41 @@ export interface IJob {
      */
     jobname: string;
 
+    /**
+     * start date of the job
+     * @type {Date}
+     * @memberof IJob
+     */
+    "exec-started": string;
+
+    /**
+     * end date of the job
+     * @type {Date}
+     * @memberof IJob
+     */
+    "exec-ended": string;
+
+    /**
+     * exec-member
+     * @type {string}
+     * @memberof IJob
+     */
+    "exec-member": string;
+
+    /**
+     * exec-submitted
+     * @type {Date}
+     * @memberof IJob
+     */
+    "exec-submitted": string;
+
+    /**
+     * exec-system
+     * @type {String}
+     * @memberof IJob
+     */
+    "exec-system": string;
+
```

Comment:
New required parameters were added to the IJob object, causing delete system tests to fail because these items must now be defined in order to delete a job. Should they be optional?